schema/data/dev/create_dev_applications.sql

```diff
@@ -0,0 +1,94 @@
+begin;
+
+  truncate ggircs_portal.application restart identity cascade;
+
+  alter table ggircs_portal.application disable trigger _send_draft_application_email;
+  alter table ggircs_portal.application_revision_status disable trigger _status_change_email;
+
+  -- Set a jwt token so that the created_by columns are not null on creation of application;
+  set jwt.claims.sub to '00000000-0000-0000-0000-000000000000';
+
+  -- Disable trigger to override default behaviour and manually assign a legacy review_step below
+  alter table ggircs_portal.application_revision_status disable trigger _create_or_refresh_review_step;
+   -- Create applications
+
+create or replace function add_dev_apps(num int)
+returns void
+as $$
+declare
+      temp_row record;
+      operator_type text;
+    begin
+      for mocked_year in 2018..2025 loop
+
+        perform mocks.set_mocked_time_in_transaction(
+          (
+            select application_open_time
+            from ggircs_portal.reporting_year
+            where reporting_year = mocked_year
+          )
+        );
+
+        if mocked_year = 2020 then
+          alter table ggircs_portal.application_revision_status enable trigger _create_or_refresh_review_step;
+        end if;
```

Comment:
Why is 2020 a special case?

Reply:
This is an attempt to create application_review_steps in the same way as they will appear in prod. So in the loop, for the years 2018 & 2019 there we create the the generic `application` review step and all years after 2020 will create review_steps from the table (currently `administrative` & `technical`) via the trigger. The trigger gets disabled at the beginning of the file & re-enabled when year=2020 in the loop (staying enabled for the rest of the following years)

Reply:
maybe let's make it `if mocked_year >= 2020` this way it's clear to the developer, and it has the same effect?

Reply:
could do, it will just run the `alter table...enable trigger...` statement a few more unnecessary times that way, but if it's clearer then the impact of that is very minimal

Reply:
I did add a comment above that line to try and be clear about what is happening